src/docfx/config/ops/OpsAccessor.cs

```diff
@@ -189,7 +181,7 @@ async Task<HttpResponseMessage> ValidationMiddleware(HttpRequestMessage request,
 
         private Task<string> FetchBuild(string urlPath, string? value404 = null, DocsEnvironment? environment = null, HttpMiddleware? middleware = null)
         {
-            return Fetch(BuildApi(environment) + urlPath, value404, BuildMiddleware(environment, middleware));
+            return Fetch(BuildApi(environment) + urlPath, value404, BuildMiddleware(middleware));
```

Comment:
It looks like a bug to remove the `environment` here. The URL is dependent on environment, but the secrets aren't?